tests/integration/helm/util.go

```diff
@@ -169,6 +169,6 @@ func VerifyInstallation(ctx framework.TestContext, cs cluster.Cluster) {
 			return fmt.Errorf("istio egress gateway pod is not ready: %v", err)
 		}
 		return nil
-	}, retry.Timeout(RetryTimeOut), retry.Delay(retryDelay))
+	}, retry.Timeout(RetryTimeOut), retry.Delay(RetryDelay))
```

Comment:
These changes are already merged in https://github.com/istio/istio/pull/33279, can you please rebase?